hnn_core/network.py

```diff
@@ -268,6 +273,12 @@ def pick_connection(net, src_gids=None, target_gids=None,
 class Network(object):
     """The Network class.
 
+    All hnn_core-models are organised in a square grid of pyramidal cells.
+    Basket cells are included at a ratio of 1:3. The separation between L2 and
+    L5 layers and the distance between grid points (i.e., pyramidal cell somas)
+    and the number of cells in the x- and y-directions are adjusted using the
+    set_cell_positions-method.
```

Comment:
I would document this in `jones_2009_model` ...

Reply:
OK to `jones_2009_model`, but `Network` will need major changes to be generic. For example, `self._params['N_pyr_x']` (and `_y`) is used, which assumes a grid. Another PR!

Reply:
agree, but one PR at a time :) Little strokes fell great oaks ...